pkg/cluster/executor/ssh.go

```diff
@@ -301,7 +301,17 @@ func (e *NativeSSHExecutor) Execute(cmd string, sudo bool, timeout ...time.Durat
 		defer cancel()
 	}
 
-	args := []string{"ssh", "-o", "StrictHostKeyChecking=no"}
+	ssh := "ssh"
+
+	isExec := func(mode os.FileMode) bool { return mode&0111 == 0111 }
+	if val := os.Getenv(localdata.EnvNameNativeSSHPath); val != "" {
+		if info, err := os.Stat(val); err == nil && !info.IsDir() && isExec(info.Mode()) {
```

Comment:
It's better to report an error if the specified binary is not existed or expected.

Reply:
sounds good